apps/crn-server/test/routes/research-outputs.route.test.ts

```diff
@@ -174,7 +174,41 @@ describe('/research-outputs/ route', () => {
       expect(response.body).toEqual(expect.objectContaining({ id: 'abc123' }));
     });
 
-    test('Should return a 500 error when creating a research output fails', async () => {
+    test('Should return a 400 error when creating a research output fails due to validation error', async () => {
+      const researchOutput = getCreateResearchOutput();
+      researchOutputControllerMock.create.mockRejectedValueOnce(
+        Boom.badRequest('error', {
+          message: 'Validation error',
+          details: ['link.iv: Another content with the same value exists.'],
+          statusCode: 400,
+        }),
+      );
+
+      const response = await supertest(app)
+        .post('/research-outputs')
+        .send({ ...researchOutput })
```

Comment:
why do we have here this destructuring?

Reply:
again - im just coping and pasting whatever it is that people used here cause unless im refactoring the whole thing i figured i'd stick to the same patterns.  It looks like the developers wanted to create a shallow clone, dont ask me why